widget/embedded/src/hooks/useWalletProviders/useWalletProviders.helpers.ts

```diff
@@ -0,0 +1,14 @@
+import type { ProviderInterface } from '@rango-dev/wallets-react';
+
+export function hashProviders(
```

Comment:
I believe another name needs to be applied.
I think that this function will return a boolean value based on its name.